src/docker-client.ts

```diff
@@ -103,9 +108,21 @@ export class DockerodeClient implements DockerClient {
     return { output, exitCode };
   }
 
-  public async buildImage(repoTag: RepoTag, context: BuildContext, buildArgs: BuildArgs): Promise<void> {
+  public async buildImage(
+    repoTag: RepoTag,
+    context: BuildContext,
+    buildArgs: BuildArgs,
+    abortBuildOnExistingImage: boolean
+  ): Promise<void> {
     log.info(`Building image '${repoTag.toString()}' with context '${context}'`);
 
+    if (abortBuildOnExistingImage) {
+      const image: Dockerode.Image = this.dockerode.getImage(repoTag.toString());
+      if (!!image.id) {
```

Comment:
Let's do an explicit check instead of coercing the value to a boolean, eg `image.id !== undefined`

Reply:
fixed ;)